File: Math/#3516-Find-Closest-Person-Easy/Solution.py
class Solution:
    def findClosestPerson(self, x: int, y: int, z: int) -> int:
        dist1 = abs(x - z)
        dist2 = abs(y - z)

        if dist1 < dist2:
            return 1
        elif dist2 < dist1:
            return 2
        else:
            return 0

File: Math/#3516-Find-Closest-Person-Easy/Solution.cpp
class Solution {
public:
    int findClosestPerson(int x, int y, int z) {
        int dist1 = abs(x - z);
        int dist2 = abs(y - z);

        if (dist1 < dist2) return 1;
        else if (dist2 < dist1) return 2;
        else return 0;
    }
};
